packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -1740,4 +1663,226 @@ describe("Garbage Collection Tests", () => {
 		assert.strictEqual(garbageCollector.tombstones.length, 0, "Expecting 0 tombstone nodes");
 		assert.strictEqual(garbageCollector.deletedNodes.size, 0, "Expecting 0 deleted nodes");
 	});
+
+	describe("GC stats", () => {
+		let garbageCollector: IGarbageCollector;
+		let initialStats: IGCStats;
+
+		const tests = (sweepEnabled: boolean) => {
+			beforeEach(() => {
+				// Set up initial GC graph with 5 nodes and 2 are unreferenced.
+				defaultGCData.gcNodes["/"] = [nodes[0]];
+				defaultGCData.gcNodes[nodes[0]] = [nodes[1]];
+				defaultGCData.gcNodes[nodes[1]] = [];
+				defaultGCData.gcNodes[nodes[2]] = [];
+				defaultGCData.gcNodes[nodes[3]] = [];
+
+				// Set up the initial GC stats based on the initial GC graph.
+				initialStats = {
+					nodeCount: 5,
+					unrefNodeCount: 2,
+					updatedNodeCount: 5,
+					dataStoreCount: 5,
+					unrefDataStoreCount: 2,
+					updatedDataStoreCount: 5,
+					attachmentBlobCount: 0,
+					unrefAttachmentBlobCount: 0,
+					updatedAttachmentBlobCount: 0,
+					lifetimeNodeCount: 5,
```

Comment:
Might be helpful for future readers to add a comment here saying these values will never change (or at least never go down - not sure if you have a test covering adding new nodes)

Reply:
The comment is already there where `tests` is called.